openlibrary/plugins/books/readlinks.py

```diff
@@ -44,7 +44,7 @@ def get_work_iaids(wkey):
     q = 'key:' + wkey
     stats.begin('solr', url=wkey)
     solr_select = solr_select_url + "?version=2.2&q.op=AND&q=%s&rows=10&fl=%s&qt=standard&wt=json&fq=type:work" % (q, filter)
-    json_data = urllib.request.urlopen(solr_select).read()
+    json_data = requests.get(solr_select).content
```

Comment:
My bet is that we want .json() (or at least str (.text)) instead of bytes (.content) in the calls in this file.  See: https://simplejson.readthedocs.io/en/latest/#simplejson.loads which says that `s` should be a str.